packs/spell-effects/spell-effect-elemental-gift-wood.json

```diff
@@ -0,0 +1,49 @@
+{
+    "_id": "qdDrSA1B4GfhhCSh",
+    "img": "systems/pf2e/icons/spells/elemental-gift.webp",
+    "name": "Spell Effect: Elemental Gift (Wood)",
```

Comment:
We should probably just condense all these

Reply:
Sounds reasonable.